### PR TITLE
Remove security contact phone field mandatory fields

### PR DESCRIPTION
**Pull request by eliagrady**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.

### CODE REVIEW

**Review by praries880 — CHANGES_REQUESTED** [review]

File: specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/security.json [hunk 1]
```diff
@@ -1179,7 +1179,6 @@
             },
             "required": [
                 "email",
-                "phone",
                 "alertNotifications",
```
[the remark this answers is not in the document]
Comment (eliagrady) [reply specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/security.json]: @praries880 I've updated the security contacts examples

File: specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/CreateSecurityContact_full_example.json [hunk 2]
```diff
@@ -18,11 +18,11 @@
...
         "200": {
             "body": {
-                "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/securityContacts/john",
-                "name": "john",
+                "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/securityContacts/default1",
+                "name": "default1",
                 "type": "Microsoft.Security/securityContacts",
                 "properties": {
-                    "email": "john@contoso.com",
+                    "email": "default1@contoso.com",
```
Comment (chlahav) [inline specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/CreateSecurityContact_full_example.json]: change back to john

File: specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/CreateSecurityContact_min_example.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        }
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/securityContacts/default2",
+                "name": "default2",
+                "type": "Microsoft.Security/securityContacts",
+                "properties": {
+                    "email": "default2@contoso.com",
```
Comment (chlahav) [inline specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/CreateSecurityContact_min_example.json]: change to a real name

File: specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/GetSecurityContact_full_example.json [hunk 4]
```diff
@@ -2,16 +2,16 @@
...
         "200": {
             "body": {
-                "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/securityContacts/john",
-                "name": "john",
+                "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/securityContacts/default1",
+                "name": "default1",
                 "type": "Microsoft.Security/securityContacts",
                 "properties": {
-                    "email": "john@contoso.com",
+                    "email": "default1@contoso.com",
```
Comment (chlahav) [inline specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/GetSecurityContact_full_example.json]: change back to john

File: specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/GetSecurityContact_min_example.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+        "securityContactName": "default2"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/securityContacts/default2",
+                "name": "default2",
+                "type": "Microsoft.Security/securityContacts",
+                "properties": {
+                    "email": "default2@contoso.com",
```
Comment (chlahav) [inline specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/GetSecurityContact_min_example.json]: use a real name

File: specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/GetSecurityContactsSubscription_full_example.json [hunk 6]
```diff
@@ -7,11 +7,11 @@
...
             "body": {
                 "value": [{
-                    "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/securityContacts/john",
-                    "name": "john",
+                    "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/securityContacts/default1",
+                    "name": "default1",
                     "type": "Microsoft.Security/securityContacts",
                     "properties": {
-                        "email": "john@contoso.com",
+                        "email": "default1@contoso.com",
```
Comment (chlahav) [inline specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/GetSecurityContactsSubscription_full_example.json]: john

File: specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/GetSecurityContactsSubscription_min_example.json [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "value": [{
+                    "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/securityContacts/default2",
+                    "name": "default2",
+                    "type": "Microsoft.Security/securityContacts",
+                    "properties": {
+                        "email": "default2@contoso.com",
```
Comment (chlahav) [inline specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/GetSecurityContactsSubscription_min_example.json]: use a real name

File: specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/UpdateSecurityContact_full_example.json [hunk 8]
```diff
@@ -16,11 +16,11 @@
...
         "200": {
             "body": {
-                "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/securityContacts/john",
-                "name": "john",
+                "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/securityContacts/default1",
+                "name": "default1",
                 "type": "Microsoft.Security/securityContacts",
                 "properties": {
-                    "email": "john@contoso.com",
+                    "email": "default1@contoso.com",
```
Comment (chlahav) [inline specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/UpdateSecurityContact_full_example.json]: change back to john

File: specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/UpdateSecurityContact_min_example.json [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+        }
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/securityContacts/default2",
+                "name": "default2",
+                "type": "Microsoft.Security/securityContacts",
+                "properties": {
+                    "email": "default2@contoso.com",
```
Comment (chlahav) [inline specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/examples/SecurityContacts/UpdateSecurityContact_min_example.json]: use a real name

**Review by chlahav — APPROVED** [review]

**Review by praries880 — APPROVED** [review]

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Pull request merged**